cmd/fsck/tui/layerbar.go

```diff
@@ -36,55 +36,84 @@ var (
 	labelStyle = lipgloss.NewStyle().Width(16).MaxWidth(16)
 
 	// stateStyle defines how the different fsck.State types show up in the progress bar.
-	stateStyle = map[fsck.State]struct {
-		// style is the style to use for this state in the progress bar.
-		style lipgloss.Style
-		// bar is the character(s) used for representing this state in the progress bar.
-		bar string
-		// priority maps each possible state to a relative importance of that priority.
-		// This is used when displaying the overall state of a range of resources where multiple
-		// states are present.
-		// The highest priority state represented in the range "wins".
-		priority int
-	}{
-		fsck.Unknown: {
+	stateStyles = []stateStyle{
+		{
+			state:    fsck.Unknown,
```

Comment:
Should this be "Scheduled" or something like this? Unknown kind of indicates that there might be something going wrong, when really, it's just that the leaf hasn't been fetched yet. Or maybe there should be a "Scheduled" AND an "Unknown" state?

Reply:
Good call, I've renamed to "Unchecked" which is what it really represents.